### PR TITLE
Percentage guidelines

### PR DESCRIPTION
**Pull request by jamiec-nd**
# Description of the Change

Added information about working with percentages in API endpoints to the CommonDataTypes.md file 

## Value
Addresses a big inconsistency we have in our API surface currently.

## Applicable Issues

Raised as part of DEKO integration.

### CODE REVIEW

File: guidelines/CommonDataTypes.md [hunk 1]
```diff
@@ -30,6 +30,26 @@ Examples
...
+```json
+{
+    "interestRate": 0.0143
+}
+```
+
+```json
+{
+    "subsidy": 0.02,
+    "subsidyDisplay": "2%"
```
Comment (alefranz) [inline guidelines/CommonDataTypes.md:49]: I'm not sold about this one. Feels like it should be the UI in control on how to render it, like with or without space.
i.e. I think according to the International System of Units it should have a space.

Would it be more flexible to have `"subsidyPercentage": 2`?

I guess in terms of what APIs should do maybe we should add more information on context. As In a Backend/Domain API should use decimal, a BFF or API Gateway could use a percentage if that is more appropriate

Comment (jamiec-nd) [reply guidelines/CommonDataTypes.md]: Updated

File: guidelines/CommonDataTypes.md [hunk 2]
```diff
@@ -30,6 +30,26 @@ Examples
 }
 ```
 
+### Percent values
+
+* Services **MUST** represent percentages as a number, where the number represents the decimal representation of the percentage. For example, `0.01` represents `1%`.
+* Services **SHOULD** provide a human-readable version of the percentage
```
Comment (alefranz) [inline guidelines/CommonDataTypes.md:36]: `Should` they really do that?
Maybe `could` based on context, like in a BFF 

something like:
```suggestion
* Services **COULD** provide a human-readable version of the percentage, if appropriate in the context of the API, clearly identifying the field with a suffix (e.g. `Percentage`, `AsPercent`, `DisplayPercent`)
```

Should we settle on the suffix? I fill in general we are quite permissive right now in these guidelines.

Comment (alefranz) [reply guidelines/CommonDataTypes.md:36]: hey @jamiec-nd , what do you think about the above comment?

Comment (jamiec-nd) [reply guidelines/CommonDataTypes.md]: @alefranz  I agree, and toyed with using the terminology `MAY` but `COULD` works for me too

**Review by alefranz — APPROVED** [review]
thanks @jamiec-nd !

File: guidelines/CommonDataTypes.md [hunk 3]
```diff
@@ -30,6 +30,26 @@ Examples
 }
 ```
 
+### Percent values
+
+* Services **MUST** represent percentages as a number, where the number represents the decimal representation of the percentage. For example, `0.01` represents `1%`.
+* Services **COULD** provide a human-readable version of the percentage, if appropriate in the context of the API, clearly identifying the field with a suffix (e.g. `Percentage`, `AsPercent`, `DisplayPercent`)
```
Comment (davidwhitney) [inline guidelines/CommonDataTypes.md:36]: Is it worth just picking one so we don't have many permutations here? We're basically saying "percentage is important enough that we kinda have a type to represent it"

**Pull request merged**